spec/models/legislator_spec.rb

```diff
@@ -227,7 +227,11 @@
 
   describe "#serializable_hash" do
     let(:senator) { FactoryGirl.create(:senator) }
-    let(:keys) { ["id", "chamber", "party", "state_rank", "name", "title", "image_url", "display_district", "state_abbrev", "district_code"] }
+    let(:keys) { [
+        "id", "chamber", "party", "state_rank", "name", "title", "image_url",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.